ruff-lint-psm-readers.toml

```diff
@@ -0,0 +1,65 @@
+# specific ruff configuraton for the psm readers (which have stricter rules)
+[lint]
+select = [
+    "ALL"
+]
+
+# TODO this is not great but it is a workaround for now:
+exclude = [
+    "**/psm_reader/*.py",
+    "**/spectral_library/reader.py",
```

Comment:
these two exclusions will go away after all rules are satisfied (that's the point of this excersise ;-))